src/chains/ethereum/ethereum/tests/api/eth/feeHistory.test.ts

```diff
@@ -0,0 +1,651 @@
+              provider,
+              gasPrice: oneGwei,
+              to,
+              from
+            });
+
+            await mineNBlocks({ provider, blocks: 1 });
+
+            const block = await provider.send("eth_getBlockByNumber", [
+              "latest"
+            ]);
+            const tx = await provider.send("eth_getTransactionByHash", [
+              txHash
+            ]);
+
+            const reward =
+              Quantity.from(tx.gasPrice).toBigInt() -
+              Quantity.from(block.baseFeePerGas).toBigInt();
+
+            const feeHistory = await provider.send("eth_feeHistory", [
+              blockCount,
+              newestBlock,
+              [10, 50, 80] // for one transaction, it will be the same for all
+            ]);
+
+            assert.deepEqual(feeHistory.reward, [[reward, reward, reward]]);
+          });
+          it("handles blocks with many transactions", async () => {
+            const blockCount = "0x1";
+            const newestBlock = "latest";
+
+            await sendLegacyTransaction({
+              provider,
+              gasPrice: oneGwei,
+              to,
+              from
+            });
+            await sendLegacyTransaction({
+              provider,
+              gasPrice: twoGwei,
+              to,
+              from
+            });
+            await sendLegacyTransaction({
+              provider,
+              gasPrice: threeGwei,
+              to,
+              from
+            });
+            await sendLegacyTransaction({
+              provider,
+              gasPrice: fourGwei,
+              to,
+              from
+            });
+
+            await mineNBlocks({ provider, blocks: 1 });
+
+            // This block has 4 standard txs, gas usage will be 84,000
+            // Each tx burns 21,000 gas in the block, making reward percentiles four equal quartiles
+            const block = await provider.send("eth_getBlockByNumber", [
+              "latest"
+            ]);
+
+            // reward = gasPrice - block.baseFeePerGas
+            const first25Reward =
+              oneGwei.toBigInt() -
+              Quantity.from(block.baseFeePerGas).toBigInt(); // reward for first 25% of block gas
+            const second25Reward =
+              twoGwei.toBigInt() -
+              Quantity.from(block.baseFeePerGas).toBigInt(); // reward for next 25% of block gas
+            const third25Reward =
+              threeGwei.toBigInt() -
+              Quantity.from(block.baseFeePerGas).toBigInt(); // etc
+            const last25Reward =
+              fourGwei.toBigInt() -
+              Quantity.from(block.baseFeePerGas).toBigInt(); // last 25%
+
+            const feeHistory = await provider.send("eth_feeHistory", [
+              blockCount,
+              newestBlock,
+              [0, 25, 26, 49.99999, 50, 50.5, 51, 75, 76, 100] // 200 is more target gas than was used, will fallback to largest reward
```

Comment:
is this an outdated comment since `200` isn't in the list?

Reply:
:+1: https://github.com/trufflesuite/ganache/pull/3351/commits/e3cf204a7281543b6f36f0c7ab6f4bd11570ea7a

Reply:
(that links to the wrong commit, but I can see that the comment was removed)